src/elements/Label/Label.js

```diff
@@ -114,12 +131,6 @@ Label.propTypes = {
   /** Additional text with less emphasis. */
   detail: PropTypes.string,
 
-  /** An element type to render the 'detail' as (string or function). */
-  detailAs: PropTypes.oneOfType([
```

Comment:
I have simplified it while [standartization](https://github.com/TechnologyAdvice/stardust/pull/452#issuecomment-243614062) is pending